v2/jam/parser/finder.go

```diff
@@ -3,14 +3,15 @@ package parser
 import (
 	"fmt"
 	"go/build"
+	"os"
 	"path/filepath"
 	"strings"
 	"time"
 
 	"github.com/gobuffalo/packr/v2/plog"
 	"github.com/karrick/godirwalk"
-	"github.com/gobuffalo/packr/v2/internal/takeon/github.com/markbates/errx"
-	"github.com/gobuffalo/packr/v2/internal/takeon/github.com/markbates/oncer"
+	"github.com/markbates/oncer"
```

Comment:
Can you please revert back to using the internal packages and not their external paths?